### PR TITLE
Add new fields contentProvider to Audio|Image|Video webhook events

### PR DESCRIPTION
**Pull request by imasahiro**
Related to #289

### CODE REVIEW

**Comment by imasahiro** [comment]
I didn't have enough time but looks if I upgrade to spring-boot2.1 (See [PR](https://github.com/line/line-bot-sdk-java/pull/280)), my change also passes integrationTest. So, let me try to review #280 first and after that, I will rebase and test this again.

File: line-bot-model/src/main/java/com/linecorp/bot/model/event/message/ContentProvider.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+     * <li> {@code line}: LINE. The binary data can be retrieved from the content endpoint.</li>
+     * <li> {@code external}: Provider other than LINE</li>
+     * </ul>
+     */
+    private final String type;
+
+    /**
+     * URL of the resource. Only included when {@link #type} is {@link #EXTERNAL}.
+     */
+    private final String originalContentUrl;
```
Comment (kazuki-ma) [inline line-bot-model/src/main/java/com/linecorp/bot/model/event/message/ContentProvider.java]: How do you think about using `URI`.
(Although it is independent topic)

Comment (imasahiro) [reply line-bot-model/src/main/java/com/linecorp/bot/model/event/message/ContentProvider.java]: I tried but other parts still uses `String` and it introduce extra data conversion at application side (String -> URI, URI -> String). I would like to apply this change in next major version.

Comment (imasahiro) [reply line-bot-model/src/main/java/com/linecorp/bot/model/event/message/ContentProvider.java]: Will address this in  https://github.com/line/line-bot-sdk-java/issues/306

**Pull request merged**